apps/portals/src/AppInitializer.tsx

```diff
@@ -15,12 +14,12 @@ const COOKIE_CONFIG_KEY = 'org.sagebionetworks.security.cookies.portal.config'
 /** On mount, update the document title and meta description using data from the portal config */
 function useSetDocumentMetadataFromConfig() {
   useEffect(() => {
-    if (document.title !== docTitleConfig.name) {
-      document.title = docTitleConfig.name
+    if (document.title !== import.meta.env.VITE_PORTAL_NAME) {
+      document.title = import.meta.env.VITE_PORTAL_NAME
     }
     document
       .querySelector('meta[name="description"]')!
-      .setAttribute('content', docTitleConfig.description)
+      .setAttribute('content', import.meta.env.VITE_PORTAL_DESCRIPTION)
```

Comment:
`docTitleConfig.json` was replaced with `.env` files loaded by Vite. This is how we access this data now.